simp_le.py

```diff
@@ -372,10 +371,11 @@ def get_output_or_fail(self, command):
     def persisted(self):
         """Call the external script and see which data is persisted."""
         output = self.get_output_or_fail('persisted')
+        persisted_keys = output.split('\n', 1)[0].split()
```

Comment:
Actually, I'd prefer current implementation to be as simple as possible, yet complying with the spec. So:
- this should be reverted to what it was before
- in future (post-#12 world, where `('key' in "account_key") is True`), we could just use `.split()`; in further future, if any kind of other metadata was passed on there, we could use your suggested code